src/AbstractOperations/interpolation_utils.jl

```diff
@@ -23,7 +23,7 @@ for ξ in ("x", "y", "z")
     end
 end
 
-instantiate(L::NTuple{N, <:DataType}) where N = Tuple(X() for X in L)
+instantiate(L::NTuple{N, <:DataType}) where N = (L[1](), L[2](), L[3]()) # Tuple(X() for X in L) 
```

Comment:
For some reason `Tuple(X() for X in L)` errors here now (see sample stacktrace below), which is why I changed it to the less general `(L[1](), L[2](), L[3]())` which should still work as all the interpolation operators have 3 dimensions and I don't think we're changing this soon.

I spent over two hours trying to figure out why `Base.Generator` errors but couldn't reproduce it in a minimal working example so I'm going to leave it in for now. I can't spend more time on this line.

---
```
Unary operations and derivatives [Float32]: Error During Test at /home/alir/Oceananigans.jl/test/test_abstract_operations.jl:243
  Test threw exception
  Expression: typeof((op(ψ))[2, 2, 2]) <: Number
  UndefVarError: I not defined
  Stacktrace:
   [1] Base.Generator(::Oceananigans.AbstractOperations.var"#5#7", ::Tuple{#s267,#s267,#s267} where #s267<:DataType) at ./generator.jl:32
   [2] instantiate(::Tuple{#s267,#s267,#s267} where #s267<:DataType) at /home/alir/Oceananigans.jl/src/AbstractOperations/interpolation_utils.jl:30
   [3] interpolation_operator(::Tuple{#s267,#s267,#s267} where #s267<:DataType, ::Tuple{DataType,DataType,DataType}) at /home/alir/Oceananigans.jl/src/AbstractOperations/interpolation_utils.jl:42
   [4] _unary_operation(::Tuple{DataType,DataType,DataType}, ::Function, ::Field{Face,Cell,Cell,OffsetArray{Float32,3,Array{Float32,3}},RegularCartesianGrid{Float32,StepRangeLen{Float64,Base.TwicePrecision{Float64},Base.TwicePrecision{Float64}}}}, ::Tuple{DataType,DataType,DataType}, ::RegularCartesianGrid{Float32,StepRangeLen{Float64,Base.TwicePrecision{Float64},Base.TwicePrecision{Float64}}}) at /home/alir/Oceananigans.jl/src/AbstractOperations/unary_operations.jl:27
   [5] sqrt(::Tuple{DataType,DataType,DataType}, ::Field{Face,Cell,Cell,OffsetArray{Float32,3,Array{Float32,3}},RegularCartesianGrid{Float32,StepRangeLen{Float64,Base.TwicePrecision{Float64},Base.TwicePrecision{Float64}}}}) at /home/alir/Oceananigans.jl/src/AbstractOperations/unary_operations.jl:92
   [6] sqrt(::Field{Face,Cell,Cell,OffsetArray{Float32,3,Array{Float32,3}},RegularCartesianGrid{Float32,StepRangeLen{Float64,Base.TwicePrecision{Float64},Base.TwicePrecision{Float64}}}}) at /home/alir/Oceananigans.jl/src/AbstractOperations/unary_operations.jl:95
   [7] top-level scope at /home/alir/Oceananigans.jl/test/test_abstract_operations.jl:243
   [8] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [9] top-level scope at /home/alir/Oceananigans.jl/test/test_abstract_operations.jl:240
   [10] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [11] top-level scope at /home/alir/Oceananigans.jl/test/test_abstract_operations.jl:231
```